content/quickstart/nodejs/metrics.md

```diff
@@ -351,20 +351,17 @@ lineReader.on("line", function (line) {
           tags,
           value: processedLine.length
         });
-
-        stats.record({
-          measure: mLatencyMs,
-          tags,
-          value: endTime.getTime() - startTime.getTime()
-        });
     } catch (err) {
-        stats.record({
-          measure: mLatencyMs,
-          {method: "repl", status: "ERROR", error: err.message},
-          value: (new Date()) - startTime.getTime()
-        });
+        tags.status = "ERROR";
+        tags.error = err.message;
     }
 
+    stats.record({
+      measure: mLatencyMs,
+      tags,
+      value: endTime.getTime() - startTime.getTime()
```

Comment:
I believe that you want `(new Date()) - startTime.getTime()` instead of `endTime.getTime() - startTime.getTime()` because `endTime` is supposed to be undefined out of the try clause.